docs/apps/components/number.md

```diff
@@ -0,0 +1,193 @@
+# Number
+
+## Description
+
+**Number** component is a UI element that allows users to input and manipulate numerical values within a form.
+
+It's commonly used for collecting quantitative data, such as age, quantity, price, or measurement values.
+
+<img src= "/apps/components/img/number.png">
+
+## Use Case
+
+1. Text Input with Number Type
+2. Number Spinner
```

Comment:
How can a number component be used for a Number Spinner or Range Slider ? This is not clear for me. Please clarify.

Reply:
4. Number Spinner: It can be used provide a user-friendly interface for inputting quantities or values. You can add custom spin buttons using JavaScript.
5. Range Slider: Based on the drag position, calculate the new min and max values for the range and update the underlying "number" element's value and attribute accordingly.